kubernetes-server-mock/src/test/java/io/fabric8/kubernetes/client/server/mock/KubernetesAttributesExtractorTest.java

```diff
@@ -84,6 +86,20 @@ public void shouldHandleResource() {
 
   }
 
+  @Test
+  public void shouldHandleResourceWithLabel() {
+    KubernetesAttributesExtractor extractor = new KubernetesAttributesExtractor();
+    Map<String, String> labels = new HashMap<>();
+    labels.put("name", "myname");
+    Pod pod = new PodBuilder().withNewMetadata().withLabels(labels).endMetadata().build();
+
+    AttributeSet attributes = extractor.extract(pod);
+
+    AttributeSet expected = new AttributeSet();
+    expected = expected.add(new Attribute("labels:name", "myname"));
+    Assert.assertTrue("Expected " + attributes + " to match " + expected, attributes.matches(expected));
```

Comment:
Just a comment: 
The project already imports AssertJ, why use JUnit assertions which are not readable at all?

Reply:
JUnit assertions are used through the rest of the file. I'm new to the project so I couldn't say why - I just aim for consistency.